python/lsst/analysis/drp/dataSelectors.py

```diff
@@ -94,7 +94,7 @@ def __call__(self, df, **kwargs):
         """
 
         result = None
-        flagCols = ["_psfFlux_flag", "_pixelFlags_saturatedCenter", "_extendedness_flag"]
+        flagCols = ["psfFlux_flag", "pixelFlags_saturatedCenter", "extendedness_flag"]
```

Comment:
I missed this one before, can you use the `flagCols` from the property `columns` here? I see the default columns listed are different... why is that? It's confusing to have a list of 4 default flag columns in one case and then override it with a different set of 3 defaults flags.